controllers/api/loginRoutes.js

```diff
@@ -1,32 +1,36 @@
 const router = require("express").Router();
 const { User } = require("../../models");
 
-// login, then redirect to user's profile page
-router.post("/login", async (req, res) => {
+router.post("/", async (req, res) => {
+  console.log("Request body:", req.body);  // Log the request body to inspect it
```

Comment:
Beware, this will show in the console of the webpage when we do the inspect for the elements